data/styles/clear/include/Basemap.mapcss

```diff
@@ -701,23 +701,25 @@ line|z18-[barrier=hedge],
 line|z18-[barrier=retaining_wall],
 line|z18-[barrier=wall],
 line|z18-[historic=citywalls],
-{width: 2.0; dashes: 2.4,2.4; opacity: 1;}
+{width: 2.0; dashes: 2.5,2.5; opacity: 1;}
 
 /* ToDo: Make linear style with small triangles pointing downwards, when drape will be ready */
-line|z16-[natural=cliff],
-line|z16-[natural=earth_bank],
-line|z16-[man_made=embankment],
-{color: @fence; width: 1.0; opacity: 0.5;}
-
-line|z17[natural=cliff],
-line|z17[natural=earth_bank],
-line|z17[man_made=embankment],
-{width: 2.0; opacity: 0.7;}
-
-line|z18-[natural=cliff],
-line|z18-[natural=earth_bank],
-line|z18-[man_made=embankment],
-{width: 3.0; opacity: 1;}
+line|z16[natural=cliff],
+line|z16[natural=earth_bank],
+line|z16[man_made=embankment],
+{color: @path; width: 1.0; opacity: 0.5;}
+
+line|z17-[natural=cliff],
+line|z17-[natural=earth_bank],
+line|z17-[man_made=embankment],
+/*
+  4 values tuple: dash (===), triangle base (tb), triangle height, base height.
+  ===\tb /===  - base height
+      \ /    | - triangle height
```

Comment:
Its not clear how "base height" is different from the thickness of the line (dash)...

Does the renderer really make breaks between the "dashes" to fit triangles into instead of just overlaying them?

I imagined it looks more like
```
================= - a line
    \/     \/     - triangles
```

Reply:
base height is the thickness of the dash line

Reply:
whys is it different from `width: 7.0;` then? (base height is just 2)

Reply:
"width" is a total pattern width = 5 + 2

Reply:
Ok, thanks, I understand now - could you please add it to the comment?

Also what (bad) will happen if `width` and `base height + triangle height` are inconsistent? Maybe its possible not to have an extra parameter if it could be derived from other params?

Is it technically hard to make it (imho) more logical like
`width` - line width w/o triangles
`dashes: dash-length, pattern-base-width, pattern-height`
?

(also I don't really like overloading the `dashes:` with something that doesn't look like a dashed line, but maybe its too much to ask)